src/pages/product-details/comments/comments.js

```diff
@@ -83,20 +86,16 @@ const Comments = ({ productId }) => {
 
   const rateTip = useMemo(() => handleRateTip(userId, language), [language, userId]);
 
-  const commentsLength = comments?.length;
-
-  const commentsList = comments
-    ? comments.map(({ _id, ...rest }) => (
-      <CommentsItem key={_id} data={rest} commentId={_id} productId={productId} />
-    ))
-    : [];
+  const commentsLength = currentLimit;
+  const commentsCount = comments.count;
```

Comment:
This also looks redundant to me.

Reply:
Current limit returns count of comments that should be shown, default is 10, and commentsCount that is all comments ex. 60. So in next formula we count how many times that button should be shown